README.md

```diff
@@ -1,3 +1,30 @@
-##### openshift-controller-manager
+# OpenShift Controller Manager
 
-TODO: Add content
+The openshift controller manager (a.k.a OCM) reconciles the status of all OpenShift API objects. The
+application consists of a main process that configures and spawns the individual controllers for
+each top-level OpenShift API object. These controllers watch their respective OpenShift API objects
+for changes and react accordingly.
+
+The controllers are generally organized by API group:
+
+- `apps.openshift.io`
+- `build.openshift.io`
+- `image.openshift.io`
+- `project.openshift.io`
+- `route.openshift.io`
+- `template.openshift.io`
+
+There are additional controllers which add OpenShift-specific capabilities to the cluster:
+
```

Comment:
If the below are going to end with a period, should they start with a capital letter?